main.go

```diff
@@ -50,6 +51,8 @@ func init() {
 		os.Exit(1)
 	}
 
+	*flagToken = os.Getenv("MASKEDEMAIL_TOKEN")
```

Comment:
This is overwriting token when passed in through the CLI with env var, so with nothing when it's not set. Precedence should be: CLI parameter > env var, if parameter is set

Reply:
Good finding, I'll add an IF clause here to ensure that it hasn't been set by CLI option.

Reply:
I've pushed a new version where I check whether the CLI token is empty and try to fetch the one from the environment. testing that commit resulted in in the desired behavior.